src/lib/utils/parser/config-validator.go

```diff
@@ -0,0 +1,106 @@
+package parser
+
+import (
+	"log"
+)
+
+// ValidateRoutesProp validates the `routes` property
+// in the configuration file.
+func ValidateRoutesProp(routes []Routes) {
+	if len(routes) == 0 {
+		log.Fatalf("`routes` property is missing.")
+	} else {
+		for i, route := range routes {
+			if route.Method == "" {
+				log.Fatalf("`method` property for route #%d is missing.", i+1)
+			}
+			if route.Route == "" {
+				log.Fatalf("`route` property for route #%d is missing.", i+1)
+			}
+			if route.URL == "" {
+				log.Fatalf("`url` property for route #%d is missing.", i+1)
+			}
+			if len(route.Header) != 0 {
+				for j, header := range route.Header {
+					if header.OfType == "" {
+						log.Fatalf("`type` property for header #%d of route `%s` is missing.", j+1, route.Route)
+					}
+					if header.Value == "" {
+						log.Fatalf("`value` property for header #%d of route `%s` is missing.", j+1, route.Route)
+					}
+				}
+			}
```

Comment:
@JayjeetAtGithub For now, I think we don't need to make `header` field mandatory. Can you please change it for now?

Reply:
I just checked and realized that `header` field is not mandatory in the above code. I am checking for validation of headers only if it is present.